src/popup/modules/QrLib/qrgen.js

```diff
@@ -99,7 +100,13 @@ export function set(tag, value) {
 export function getQr() {
     console.info("generated new QrGen qr code");
 
-    const qrElem = QRC.encodeText(qrText, qrErrorCorrection);
+    try {
+        const qrElem = QRC.encodeText(qrText, qrErrorCorrection);
```

Comment:
and here eslint also catches a logic error that could result in problems/bugs in certain cases 😄